egs/wenetspeech/ASR/pruned_transducer_stateless/asr_datamodule.py

```diff
@@ -361,10 +363,15 @@ def test_dataloaders(self, cuts: CutSet) -> DataLoader:
         sampler = DynamicBucketingSampler(
             cuts, max_duration=self.args.max_duration, shuffle=False
```

Comment:
You must set `rank=0, world_size=1` for the sampler, otherwise you might be dropping some data. There should be a big warning about this in your run logs.

Reply:
Hmm that's possible if you used a single-GPU process to decode.

Reply:
FYI I was mentioning this warning:

https://github.com/lhotse-speech/lhotse/blob/dc0c86bd498dd089b2165673d150b95bcbc0f768/lhotse/dataset/iterable_dataset.py#L66-L73